### PR TITLE
Webpage Layout For Desktop and Mobile Version

### PR DESCRIPTION
**Pull request by JbirdL86**
This branch adds HTML and styling to both, the index page and the about page. Also, there are 2 JS files for DOM manipulation. The nav.js file is for user interactions and the speakers.js works to populate the speakers' section. The files containing all these changes are:

- index.html
- about.html
- nav.js
- speakers.js

You can click on this link https://www.loom.com/share/edfdcd8e87ed46459854f749120c0454 to watch a video describing my project.

I hope you like my project! +1

### CODE REVIEW

File: speakers.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+function createSpeakers() {
+  let speaker;
+
+  speaker = new Speaker('Bill Gates', 'Software developer, investor, entrepreneur', 'William Henry Gates III is an American business magnate, software developer, investor, author, and philanthropist.', 'assets/billgates.jpeg');
+  speakers.push(speaker);
+
+  speaker = new Speaker('Elon Musk', 'Founder, CEO and Chief Engineer of SpaceX', 'Elon Reeve Musk is an entrepreneur and business magnate. He is the founder, CEO, and Chief Engineer at SpaceX; early stage investor, CEO, and Product Architect of Tesla, Inc.', 'assets/elonmusk.png');
+  speakers.push(speaker);
+}
```
Comment (Shaher-11) [inline speakers.js]: Add more speakers  so you could enable this  wrong2 button

File: speakers.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+function Speaker(name, profession, description, featureImg) {
+  this.name = name;
+  this.profession = profession;
+  this.description = description;
+  this.featureImg = featureImg;
+}
+
+const speakers = [];
```
Comment (Shaher-11) [inline speakers.js:8]: Increase the speaker number as shown in the original design below
#### Original design
right1

#### Your Project
wrong1

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @JbirdL86,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured and the readme file is great:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @JbirdL86  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

As Optional, you could add different images for the speakers.

Good job on your project!

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**